Userland/Libraries/LibSanitizer/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+set(SOURCES
+    UndefinedSanitizer.cpp
+)
+
+serenity_lib(LibSanitizer sanitizer)
+target_link_libraries(LibSanitizer LibC)
```

Comment:
Maybe `LibSanitizer` shoud just be part of `LibC`, so we don't need to go make it a dependency everwhere?
Or we could just inject the depenedency in the `serenity_bin(..)` / `serenity_app(..)` cmake functions? 

It might also be nice to add a new `option(ENABLE_USER_SPACE_UBSAN, ...)` in the top level CMakeLists.txt defauling to ON, so folks can control this if they want?

Reply:
What great timing as the CI failure is due to something just like this (new app in master not having LibSanitizer as a dep).

Putting it in LibC feels weird to me for some reason(?) but I am so far removed from an expert to make a decision on something like that. At the very least making it a dependency for programs automatically very much seems like a good idea. I will also add in the requested option :)